docs-user/guide-ui-tour.md

```diff
@@ -32,7 +32,7 @@ Committing a range selection by clicking the zoom button is a useful way to zoom
 
 Each thread in the timeline contains a thread stack graph. The X axis represents time, and the Y axis represents the height of the stack. It's important to realize that the height does not correlate with the amount of work that was being done. It only represents the arbitrary height of the stack at the time. It can be used as a useful way to get oriented with what was executing at the time.
 
-Clicking on the thread stack graph will select that stack and update the panel below (if the panel uses the currently selected stack). For instance, in the call tree, it will open up the tree to find the stack. The selected stack will be colored a darker blue in the list. This is another useful way to correlate stacks with when they were executed over time.
+Clicking on the thread stack graph will select that stack and update the panel below (if the panel uses the currently selected stack). For instance, in the call tree, it will open up the tree to find the stack. The selected stack will be darker blue in the list. This is another useful way to correlate stacks with when they were executed over time.
```

Comment:
This change isn't needed with `profanitySureness: 2`, it's only shown with 1 or 0 (I don't remember which one).